validator/src/commands/run/execute.rs

```diff
@@ -1044,6 +1044,17 @@ pub fn execute(
         "block_verification_method",
         BlockVerificationMethod
     );
+    match validator_config.block_verification_method {
```

Comment:
i think we should have also `.hide_possible_values(true)` on the `clap::Arg` and added the remaining valid value to `.help()` statically

Reply:
That isn't something we have done in the past, but I see some merit to your comment. Do you view this as a blocker for merging this PR tho ?

Reply:
we have continued to make deprecated arg values discoverable in the past?

Reply:
Here is a similar PR for when we started to retire `tlmi`: https://github.com/anza-xyz/agave/pull/3113